decode/layout.ts

```diff
@@ -152,11 +159,12 @@ function getHash(id: number, tag: string, path: string, attributes: Attributes):
         case "*D":
             break;
         default:
+            tag = tag.indexOf(SVG_PREFIX) === 0 ? tag.substr(SVG_PREFIX.length) : tag;
             let s = path && path.length > 0 ? path + tag : tag;
```

Comment:
Isn't this repeating logic for computing the selector? Can you move it one location?

Reply:
Echoing this comment again. Similarly looking logic in two places. Refactor. 

---
In reply to: [334058330](https://github.com/microsoft/clarity-js/pull/151#discussion_r334058330) [](ancestors = 334058330)

Reply:
Done. I left the comment open because I wanted to get back to it and was on my list as well. It was getting redundant to write same things at two different places and keep them in sync. We can now expand on selector logic without worry about encoding / decoding.